src/test/java/pl/betse/beontime/LoginControllerTest.java

```diff
@@ -0,0 +1,80 @@
+package pl.betse.beontime;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import pl.betse.beontime.bo.UserDTO;
+import pl.betse.beontime.controller.LoginController;
+import pl.betse.beontime.entity.DepartmentEntity;
+import pl.betse.beontime.entity.UserEntity;
+import pl.betse.beontime.model.exception.UserBadCredentialException;
+import pl.betse.beontime.service.UsersService;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.when;
+
+class LoginControllerTest {
+
+    private UsersService userService = Mockito.mock(UsersService.class);
+    private PasswordEncoder passwordEncoder = Mockito.mock(BCryptPasswordEncoder.class);
+    private LoginController loginController;
+    private UserDTO TEST_USER;
+    private UserEntity TEST_USER_ENTITY;
+
+
+    @BeforeEach
+    void setup() {
+        TEST_USER = new UserDTO("CustomGUID", "test@test.pl", "TestName", "TestLastName", false, "BANKING", new ArrayList<>());
+        TEST_USER_ENTITY = new UserEntity(1, "CustomGUID", TEST_USER.getEmail(), TEST_USER.getFirstName(), TEST_USER.getLastName(), "PASSWORD", TEST_USER.isActive(), new DepartmentEntity(1, "BANKING"), new HashSet<>());
+        loginController = new LoginController(userService, passwordEncoder);
+    }
+
+
+    @Test
+    void checkExsistsByEmailLogin() {

```

Comment:
test should start with ``should`` and describe what tested method should return or do